src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,106 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double DEFAULT_RESIZE_VALUE = 1.5;
+    private int changeableCapacity;
```

Comment:
You don't need this variable. Use `elements.length` if you need to know current size